src/components/CarDetails/CarDetails.jsx

```diff
@@ -66,7 +66,11 @@ const CarDetails = ({ car }) => {
         </div>
 
         <div className="text-center my-2">
-          <button className="btn btn-dark px-5 text-white" type="button">
+          <button
+            className="btn btn-dark px-5 text-white"
+            type="button"
+            onClick={() => navigate('/reserve')}
+          >
```

Comment:
The reserve button should autofill the selected car when clicked from here and should only ask for start date, end date and city
It behaves correctly when the reserve model is clicked from the navigation.